server/server.go

```diff
@@ -66,6 +68,7 @@ type config struct {
 	Port             string `envconfig:"PORT" default:"6789"`
 	ReportPath       string `envconfig:"REPORT_PATH" default:""`
 	PathPrefix       string `envconfig:"KEPLOY_PATH_PREFIX" default:"/"`
+	LogPath          string `envconfig:"LOG_PATH" default:"logfile"`
```

Comment:
Also we dont need a default value here, because by default logs should logged in console.